specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -117,7 +117,8 @@
       "properties": {
         "activity": {
           "description": "Activity name.",
-          "type": "string"
+          "type": "string",
+          "minLength": 1
```

Comment:
this is a breaking change. Looks like this restriction is added in the existing api-version.